lib/metadata.js

```diff
@@ -28,11 +28,13 @@ import { NOT_KNOWN } from './constants';
 function determineMetadata(data) {
     let instanceData;
     const currentCapabilities = data.capabilities;
+    const { capabilities: optsCaps = {}, requestedCapabilities = {} } = browser.options || {}
     const currentConfigCapabilities = data.config.capabilities || {};
-    const metadata = currentConfigCapabilities['cjson:metadata']
-        // For WDIO V6
-        || browser.options.requestedCapabilities.w3cCaps.alwaysMatch['cjson:metadata']
-        || {};
+    const { w3cCaps = {} } = requestedCapabilities;
+    const metadata = currentConfigCapabilities['cjson:metadata'] // For WDIO V6
+    || (w3cCaps.alwaysMatch && w3cCaps.alwaysMatch['cjson:metadata']) // When an app is used to test
+    || optsCaps['cjson:metadata']; // devtools
```

Comment:
```suggestion
    || optsCaps['cjson:metadata'] // devtools
```

Reply:
This breaks the module, can you also add a test for it?

Reply:
Sorry about that, should be good to go.